http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/SecurityFilesConfig.scala

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.dsde.workbench.leonardo.config
+
+import java.nio.file.Path
+
+case class SecurityFilesConfig(proxyServerCrt: Path,
```

Comment:
will this be shared by both kubernetes app and jupyter/rstudio?

Reply:
yeah, they use the same crts